src/workerd/io/compatibility-date.capnp

```diff
@@ -679,4 +679,9 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
       $compatDisableFlag("cache_no_cache_disabled")
       $experimental;
   # Enables the use of cache: no-cache in the fetch api.
+
+  requestCfOverridesCacheRules @71 :Bool
+      $compatEnableFlag("request_cf_overrides_cache_rules")
+      $experimental;
```

Comment:
Does this also need the `$neededByFl` flag, so that it automatically gets sent to FL in the subrequest?